frame/support/procedural/src/lib.rs

```diff
@@ -325,3 +325,34 @@ pub fn construct_runtime(input: TokenStream) -> TokenStream {
 pub fn transactional(attr: TokenStream, input: TokenStream) -> TokenStream {
 	transactional::transactional(attr, input).unwrap_or_else(|e| e.to_compile_error().into())
 }
+
+/// Assert the annotated function is executed within a storage transaction.
```

Comment:
Please move the docs above the re-export in `frame-support` and make the example compile. Having compiling examples is much better, because we will be informed when something breaks ;)

Reply:
Do I need to move the docs if the re-export already have `#[doc(inline)]`?

https://github.com/paritytech/substrate/blob/7b206944217fa61acff88369570e4b7b4400722d/frame/support/src/lib.rs#L269-L272